src/redux/rockets/rocketsSlice.js

```diff
@@ -0,0 +1,51 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import api from '../../api/api';
+
+// Actions
+const GET_ROCKETS = 'GET_ROCKETS';
+
+const initialState = {
+  rockets: [],
+  status: 'idle', // 'idle' | 'loading' | 'succeeded' | 'failed'
+  error: null,
+  msg: null,
+};
+
+// Thunks
+export const getRockets = createAsyncThunk(GET_ROCKETS, async () => {
+  try {
+    return await api.fetchRockets();
+  } catch (error) {
+    return error.message;
+  }
+});
+
+// Reducer
+const rocketsSlice = createSlice({
+  name: 'rockets',
+  initialState,
+  reducers: {},
+  extraReducers: (builder) => {
+    builder
+      .addCase(getRockets.pending, (state) => ({
+        ...state,
+        status: 'loading',
+      }))
+      .addCase(getRockets.fulfilled, (state, action) => ({
+        ...state,
+        rockets: action.payload,
+        status: 'succeeded',
+      }))
+      .addCase(getRockets.rejected, (state, action) => ({
+        ...state,
+        status: 'failed',
+        error: action.error.message,
+      }));
+  },
+});
+
+export const allRockets = (state) => state.rockets.rockets;
+export const allStatus = (state) => state.rockets.status;
+export const allMsgs = (state) => state.rockets.msg;
```

Comment:
[Optional] Similar to my previous recommendations. It is amazing using the CamelCase for naming. Consider using `allMessages` instead of `allMsgs` and `state.rockets.messages` instead of `state.rockets.msg` for better readability and consistency.

The increased text content may not be significantly longer

Reply:
I will change it like so thanks